### PR TITLE
Add swagger-ui to document openAPI

### PR DESCRIPTION
**Pull request by zloery**
This adds a swagger-ui page at /api/swagger-ui.html that users can use to view and run APIs.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -53,7 +54,7 @@ configurations.all {
 // - define the validation
 // - define dependencies to include the generated code
 
-def openapiSourceFile = "${projectDir}/src/main/resources/service_openapi.yaml"
+def openapiSourceFile = "${projectDir}/src/main/resources/api/service_openapi.yaml"
```
Comment (ddietterich) [inline build.gradle:57]: Why that change?

**Review by ddietterich — APPROVED** [review]
One minor question.
LGTM

File: build.gradle (hunk 1 above)
Comment (zloery) [reply build.gradle:57]: I wanted to organize the resources/ directory a bit since it had both the API yaml and the UI html page. Either way works

Comment (ddietterich) [reply build.gradle:57]: That makes sense. I was worried it was some weird "you have to put it here" artifact of the doc generation.

**Pull request merged**